languages/en.ini

```diff
@@ -893,6 +893,7 @@ Other Sources = "Other Sources"
 other_versions_link = "Show other versions (%%count%%)"
 other_versions_search_link = "Show all versions (%%count%%)"
 other_versions_title = "Other Versions (%%count%%)"
+outdated_browser_message_html = "<h2>Your are using outdated and unsupported browser.</h2><p>To ensure information pricacy and full functionality of site, we recommend you update your browser.</p>"
```

Comment:
Typo: "Your are" instead of "You are". Might also be good to add "the" in the phrase "full functionality of site".

Reply:
Oh, thanks!

Reply:
This one has been now addressed